docs/dev-guide/src/getting-started/building.md

```diff
@@ -13,11 +13,14 @@ not by installing your distro's `rust` package which will most likely be too old
 
 For the installation you can refer to [these instructions](https://www.rust-lang.org/tools/install).
 
-Once you have a working rust installation, you'll need to add the `wasm32-unknown-unknown` target and install the nightly toolchain
+Once you have rustup, you'll need to do the following.
+1. Install rust version 1.76.0. Creditcoin uses the feature `stdsimd` which was removed in later Rust versions.
+2. Default to using the nightly toolchain corresponding to Rust version 1.76.0 as shown below.
+3. Add the `wasm32-unknown-unknown` target
 
 ```bash
-rustup update
-rustup toolchain install nightly
+rustup install 1.76.0
+rustup install nightly-2024-02-04
```

Comment:
The actual version is specified in `ci/env` so you may want to refer to that file and update it instead of hard-coding a different version in README.

Reply:
Because of how the compiler version is specified you may want to add a step which makes the nightly version the default one, or better yet add `rust-toolchain.toml` file to replace `ci/env`. 

See https://github.com/gluwa/creditcoin3/blob/dev/.github/workflows/ci.yml#L202 as well b/c the GitHub Actions that we have don't understand this file correctly, eventhough they are supposed to.